stat/it/entropy_test.go

```diff
@@ -0,0 +1,81 @@
+package it_test
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
```

Comment:
This needs to be "golang.org/x/exp/rand" (this is the cause of the second failure).